src/webserver/REPLTools.jl

```diff
@@ -117,9 +117,9 @@ responses[:docs] = function response_docs(🙋::ClientRequest)
         query = "@$(query[begin:end-1])_str"
     end
 
-    doc_html, status = if REPL.lookup_doc(Symbol(query)) isa Markdown.MD
+    doc_html, status = if (doc_md = Docs.doc(Docs.Binding(Base, Symbol(query)))) isa Markdown.MD &&
```

Comment:
This local fetcher was never actually working (it is now), so maybe we can just get rid of it ?